bug: opencode-subagent-statusline@0.9.0 fails with ERESOLVE on @opentui/core peer conflict

## Summary

opencode-subagent-statusline@0.9.0 fails to install/upgrade when @opentui/core@0.4.0 is present due to a peer dependency conflict. The plugin's package.json declares @opentui/core@">=0.2.10 <0.4", which excludes the 0.4.0 release already required by the OpenCode plugin stack.

This error was exposed while upgrading gentle-ai from 1.36.7 to 1.37.0 using the TUI **Upgrade Tools** flow, which offered the plugin update from 0.8.0 to 0.9.0. The same failure can be reproduced manually with npm upgrade.

## Steps to Reproduce

1. Have OpenCode 1.16.2 installed (pulls @opencode-ai/plugin@1.16.2 and @opentui/core@0.4.0)
2. Have opencode-subagent-statusline@0.8.0 installed
3. Run gentle-ai 1.37.0 and open **Upgrade Tools**
4. When the update opencode-subagent-statusline@0.8.0 → 0.9.0 is offered, confirm the upgrade
5. Observe npm error ERESOLVE unable to resolve dependency tree

## Expected Behavior

opencode-subagent-statusline@0.9.0 should install/upgrade successfully from 0.8.0 on systems running @opentui/core@0.4.0.

## Actual Behavior

npm aborts with:

```bash
npm error ERESOLVE unable to resolve dependency tree
npm error Could not resolve dependency:
npm error peer @opentui/core@">=0.2.10 <0.4" from opencode-subagent-statusline@0.9.0
npm error node_modules/opencode-subagent-statusline
npm error   opencode-subagent-statusline@"0.9.0" from the root project
npm error Found: @opentui/core@0.4.0
npm error node_modules/@opentui/core
npm error   peerOptional @opentui/core@">=0.3.2" from @opencode-ai/plugin@1.16.2
npm error   @opentui/core@"0.4.0" from @opentui/keymap@0.4.0
```

The upgrade of gentle-ai itself completes successfully (1.36.7 → 1.37.0), but the plugin upgrade (0.8.0 → 0.9.0) fails with "Summary: 1 upgraded, 1 failed". The same error occurs running npm upgrade opencode-subagent-statusline manually in ~/.config/opencode.

## Environment

- OS: Linux (Ubuntu/Debian)
- Node.js version: 22.x
- pnpm version: N/A (npm invoked by gentle-ai upgrade)
- OpenCode version: 1.16.2
- @opencode-ai/plugin: 1.16.2
- @opentui/core installed: 0.4.0
- opencode-subagent-statusline current version: 0.8.0
- opencode-subagent-statusline target version: 0.9.0
- gentle-ai version where the issue appeared: 1.37.0
- Previous gentle-ai version: 1.36.7 (the plugin update to 0.9.0 was not offered / did not fail there)

## Additional Context

The peerDependencies in the plugin's package.json (current main) are:

```json
"peerDependencies": {
  "@opencode-ai/plugin": ">=1.14.50 <2",
  "@opentui/core": ">=0.2.10 <0.4",
  "@opentui/solid": ">=0.2.10 <0.4",
  "solid-js": ">=1.9.12 <2"
}
```

Both @opencode-ai/plugin@1.16.2 and @opentui/keymap@0.4.0 already depend on @opentui/core@0.4.0, so the <0.4 upper bound is now incompatible with the current OpenCode release line.

**Suggested fix**: bump the peer dependency ranges to allow @opentui/core@^0.4.0 and @opentui/solid@^0.4.0, e.g.:

```json
"@opentui/core": ">=0.2.10 <0.5",
"@opentui/solid": ">=0.2.10 <0.5"
```

There is also a related issue in the gentle-ai installer (https://github.com/Gentleman-Programming/gentle-ai/issues/793), but the root cause is the peer range here.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,7 +17,7 @@ jobs:
       - name: Setup pnpm
         uses: pnpm/action-setup@v4
         with:
-          version: 11.1.2
+          version: 11.2.2
 
       - name: Setup Node.js
         uses: actions/setup-node@v4

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -28,7 +28,7 @@ jobs:
       - name: Setup pnpm
         uses: pnpm/action-setup@v4
         with:
-          version: 11.1.2
+          version: 11.2.2
 
       - name: Setup Node.js
         uses: actions/setup-node@v4

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "https://github.com/Joaquinvesapa/sub-agent-statusline"
   },
-  "packageManager": "pnpm@11.1.2",
+  "packageManager": "pnpm@11.2.2",
   "engines": {
     "node": ">=22.13"
   },
@@ -57,25 +57,25 @@
   "license": "MIT",
   "peerDependencies": {
     "@opencode-ai/plugin": ">=1.14.50 <2",
-    "@opentui/core": ">=0.2.10 <0.4",
-    "@opentui/solid": ">=0.2.10 <0.4",
+    "@opentui/core": ">=0.4.0 <0.5",
+    "@opentui/solid": ">=0.4.0 <0.5",
     "solid-js": ">=1.9.12 <2"
   },
   "devDependencies": {
-    "@opencode-ai/plugin": "^1.14.50",
-    "@opentui/core": "^0.2.10",
-    "@opentui/solid": "^0.2.10",
+    "@opencode-ai/plugin": "^1.15.11",
+    "@opentui/core": "^0.4.0",
+    "@opentui/solid": "^0.4.0",
     "@semantic-release/commit-analyzer": "^13.0.1",
     "@semantic-release/github": "^11.0.6",
     "@semantic-release/npm": "^12.0.2",
-    "@semantic-release/release-notes-generator": "^14.1.0",
-    "@types/node": "^22.15.21",
-    "@vitest/coverage-v8": "^4.1.5",
+    "@semantic-release/release-notes-generator": "^14.1.1",
+    "@types/node": "^22.19.19",
+    "@vitest/coverage-v8": "^4.1.7",
     "esbuild-plugin-solid": "^0.6.0",
     "semantic-release": "^24.2.9",
     "tsup": "^8.5.1",
-    "typescript": "^5.8.3",
-    "vitest": "^4.1.5"
+    "typescript": "^5.9.3",
+    "vitest": "^4.1.7"
   },
   "release": {
     "branches": [

--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -1,0 +1,11 @@
+minimumReleaseAgeExclude:
+  - '@opentui/core-darwin-arm64@0.4.0'
+  - '@opentui/core-darwin-x64@0.4.0'
+  - '@opentui/core-linux-arm64-musl@0.4.0'
+  - '@opentui/core-linux-arm64@0.4.0'
+  - '@opentui/core-linux-x64-musl@0.4.0'
+  - '@opentui/core-linux-x64@0.4.0'
+  - '@opentui/core-win32-arm64@0.4.0'
+  - '@opentui/core-win32-x64@0.4.0'
+  - '@opentui/core@0.4.0'
+  - '@opentui/solid@0.4.0'

